client/modules/IDE/components/About.jsx

```diff
@@ -26,15 +26,15 @@ function About(props) {
         </p>  */}
       </div>
       <div className="about__content-column">
-        <h3 className="about__content-column-title">{t('NewP5')}</h3>
+        <h3 className="about__content-column-title">{t('About.About_Modal.NewP5')}</h3>
```

Comment:
I wonder if the `About_Modal` is needed as there's a bit of duplication. Could it just be `About.Modal.NewP5`?

I'm not sure though, what do you think?

Reply:
Yes, looks better and less letters = less chances for typos